src/System.Private.ServiceModel/src/System/ServiceModel/Channels/CoreClrSocketConnection.cs

```diff
@@ -585,7 +592,7 @@ protected override int ReadCore(byte[] buffer, int offset, int size, TimeSpan ti
             catch (ObjectDisposedException objectDisposedException)
             {
                 Exception exceptionToThrow = ConvertObjectDisposedException(objectDisposedException, TransferOperation.Read);
-                if (object.ReferenceEquals(exceptionToThrow, objectDisposedException))
```

Comment:
Nit: For future consideration, since we decided to do an all-up CodeFormatter pass, I'd prefer not to include Resharper cleanup in the PRs.  It contributes to the noise to review.